src/me/remie/vulcan/leaguetasks/utils/LeaguePanel.java

```diff
@@ -0,0 +1,123 @@
+package me.remie.vulcan.leaguetasks.utils;
+
+import simple.hooks.simplebot.Game;
+import simple.hooks.wrappers.SimpleWidget;
+import simple.robot.api.ClientContext;
+
+public class LeaguePanel {
+    private ClientContext ctx;
```

Comment:
```suggestion
    private final ClientContext ctx;
```
nitpick from previous code review